BAZProjectC1/MovieAPI.swift

```diff
@@ -31,5 +31,36 @@ class MovieAPI {
 
         return movies
     }
-
+    
+    func getMoviesUpdate(completion: @escaping ([Movie]) -> ()) {
+        guard let url = URL(string: "https://api.themoviedb.org/3/trending/movie/day?api_key=\(apiKey)") else {
+            return
+        }
+        
+        let session = URLSession.shared
+        session.dataTask(with: url) { (data, response, error) in
+            guard let data = data else {
+                return
+            }
+            
+            do {
+                let json = try JSONSerialization.jsonObject(with: data, options: [])
+                completion(self.serializationJson(jsonDic: json))
+            } catch {
+                completion([])
+            }
+        }.resume()
+        
+    }
+    
+    //MARK: Serializacion
+    func serializationJson(jsonDic: Any) -> [Movie] {
+        var lstMovies = [Movie]()
+        if let dataInfo = jsonDic as? [String : Any], let dataAll = dataInfo["results"] as? [[String: Any]]{
```

Comment:
Deja un espacio entre el agrumento y la llave
```suggestion
        if let dataInfo = jsonDic as? [String : Any], let dataAll = dataInfo["results"] as? [[String: Any]] {
```